package.json

```diff
@@ -64,10 +66,14 @@
     "@types/superagent": "^8.1.7",
     "chai": "^5.1.0",
     "coveralls": "^3.1.1",
+    "eslint": "^9.3.0",
+    "eslint-config-prettier": "^9.1.0",
```

Comment:
you may not need this anymore since eslint 9 disabled most or all stylistic rules. worth trying a lint without it